core/Filesystem.php

```diff
@@ -161,9 +162,11 @@ public static function checkIfFileSystemIsNFS()
             $output = @shell_exec($command);
             if ($output) {
                 $commandFailed = (false !== strpos($output, "no file systems processed"));
-                $output = explode("\n", trim($output));
+                $output = trim($output);
+                $outputArray = explode("\n", $output);
                 if (!$commandFailed
-                    && count($output) > 1) {
+                    && count($outputArray) > 1
+                    && strpos(' nfs', $output) !== false) {
```

Comment:
Hi @Seb35, instead of looking for a space prefix, can we check for a word boundary using a regex? Just in case the output has tabs.

Reply:
I changed the condition to `preg_match('/bnfs\d?\b/', $output)` to search "nfs", "nfs4" or any "nfs[0-9]" as a single word.